ui/app/AppLayouts/Profile/views/PrivacyAndSecurityView.qml

```diff
@@ -38,13 +30,15 @@ SettingsContentBase {
                 StatusSwitch {
                     id: enableMetricsSwitch
                     checked: root.isCentralizedMetricsEnabled
-                    onClicked: {
-                        Global.openMetricsEnablePopupRequested(Constants.metricsEnablePlacement.privacyAndSecurity, popup => popup.toggleMetrics.connect(refreshSwitch))
+                    onToggled: {
+                        Global.openMetricsEnablePopupRequested(Constants.metricsEnablePlacement.privacyAndSecurity, null)
```

Comment:
please remove to callback parameter from `Global.openMetricsEnablePopupRequested`. It's not used at all. Similarly in `main.qml`, where the signal is handled.